packages/react-api/src/Api.tsx

```diff
@@ -23,6 +23,7 @@ import addressDefaults from '@polkadot/util-crypto/address/defaults';
 
 import ApiContext from './ApiContext';
 import registry from './typeRegistry';
+import './initSettings';
```

Comment:
Not 100% sure this is 100% the right place, but let's keep it here, don't have better ideas atm.